spec/system/dashboard/users_spec.rb

```diff
@@ -58,10 +61,13 @@
       click_button 'Edit'
 
       choose('Admin')
+      select location.name_en, from: 'Locations'
       click_button('Update User')
-      admin.reload
+      user.reload
 
       expect(page).to have_text('User was successfully updated.')
+      expect(user.roles?(:admin)).to eq true
+      expect(user.location_ids).to include location.id
```

Comment:
FYI, usually we don't go to check database level because it's internal structure of application and it's an implementation detail, but it's okay for now.